cachi2/core/package_managers/bundler/parser.py

```diff
@@ -103,10 +105,6 @@ def download_to(self, deps_dir: RootedPath) -> None:
         log.info(
             "Downloading platform-specific gem %s-%s-%s", self.name, self.version, self.platform
         )
-        # A combination of Ruby v.3.0.7 and some Bundler dependencies results in
-        # -gnu suffix being dropped from some platforms. This was observed on
-        # sqlite3-aarch-linux-gnu. We discourage using outdated platforms
-        # for building dependencies and cnsider this to be a limitation of Ruby.
```

Comment:
If you want to remove this hunk, you need to do it in a separate commit and describe the motivation accordingly.

Reply:
I thought it was a leftover from the last bundler PR.

Reply:
Please don't. This is not a leftover, this is intentional. Figuring out that certain combinations cause interesting failures was rather time-consuming and I had to document that somewhere. This spot is close enough to a place in code that could suffer from the issue, so I chose it.